addons/account/models/account_move.py

```diff
@@ -1648,30 +1648,6 @@ def _onchange_journal_id(self):
     # CONSTRAINT METHODS
     # -------------------------------------------------------------------------
 
-    @api.constrains('name', 'journal_id', 'state')
```

Comment:
So... There is no constraint anymore?

Reply:
We would not strictly need it anymore, but we can still add it with the exception that multiple moves without number yet are allowed in posted state.

Reply:
> We would not strictly need it anymore

![uh?](https://pbs.twimg.com/profile_images/1257565005391835136/tRjpU4IU_400x400.jpg)